index.js

```diff
@@ -3842,6 +3842,38 @@
   S.div =
   def('div', {}, [$.FiniteNumber, $.NonZeroFiniteNumber, $.FiniteNumber], div);
 
+  //# pow :: FiniteNumber -> (FiniteNumber -> FiniteNumber)
+  //.
+  //. Takes a finite number `n` and returns the _`n` squared_ function.
```

Comment:
This should read _power of `n`_ rather than _`n` squared_. I'll fix this in the related pull request on which I'm currently working.